### PR TITLE
fix(authority): allow raw domains to be set

### PR DESCRIPTION
**Pull request by stakach**
fixes placeos/rest-api#127

### CODE REVIEW

File: spec/authority_spec.cr [hunk 1]
```diff
@@ -15,6 +15,9 @@ module PlaceOS::Model
       authority = Authority.new
       authority.domain = "https://localhost:8080"
       authority.domain.should eq "localhost"
+
+      authority.domain = "test.somewhere.com"
+      authority.domain.should eq "test.somewhere.com"
```
Comment (caspiano) [inline spec/authority_spec.cr:20]: this also works

```crystal
 URI.parse("/wrong").path # => “/wrong”
```

Comment (stakach) [reply spec/authority_spec.cr:20]: yep, but that is fine, it won't break anything if someone enters that - basically the same as entering the wrong domain

Comment (stakach) [reply spec/authority_spec.cr:20]: like it I enter `my.doman` instead of `my.domain` it has the same impact as entering `/obviously-wrong`

Comment (caspiano) [reply spec/authority_spec.cr:20]: do we want to add validation? or is it not worth it?

Comment (stakach) [reply spec/authority_spec.cr:20]: mostly just not sure it's needed, 0 impact if they enter the wrong thing outside of their domain not working and they can enter the wrong thing even with extensive validation
can't see any malicious vectors as all we do is compare this with a host header

**Review by caspiano — APPROVED** [review]

**Pull request merged**